src/renderer/components/watch-video-comments/watch-video-comments.js

```diff
@@ -148,26 +128,49 @@ export default Vue.extend({
     },
 
     getCommentReplies: function (index) {
-      switch (this.commentData[index].dataType) {
-        case 'local':
-          this.getCommentRepliesLocal({
-            videoId: this.id,
-            setCookie: false,
-            sortByNewest: this.sortNewest,
-            replyToken: this.commentData[index].replyToken,
-            index: index
-          })
-          break
-        case 'invidious':
-          this.getCommentRepliesInvidious(index)
-          break
+      if (process.env.IS_ELECTRON) {
+        switch (this.commentData[index].dataType) {
+          case 'local':
+            this.getCommentRepliesLocal(index)
+            break
+          case 'invidious':
+            this.getCommentRepliesInvidious(index)
+            break
+        }
+      } else {
+        this.getCommentRepliesInvidious(index)
       }
     },
 
-    getCommentDataLocal: function (payload) {
-      ytcm.getComments(payload).then((response) => {
-        this.parseLocalCommentData(response, null)
-      }).catch((err) => {
+    getCommentDataLocal: async function (more) {
+      try {
+        /** @type {import('youtubei.js/dist/src/parser/youtube/Comments').default} */
+        let comments
+        if (more) {
+          comments = await this.nextPageToken.getContinuation()
+        } else {
+          comments = await getLocalComments(this.id, this.sortNewest)
+        }
+
+        const parsedComments = []
+        for (const commentThread of comments.contents) {
+          const hasReplies = commentThread.has_replies
+          const hasOwnerReplied = hasReplies ? commentThread.comment_replies_data.has_channel_owner_replied : false
```

Comment:
Is there a reason this isn't part of the parseLocalComment function?

Reply:
If I had a reason I can't remember it, so I've moved it into the function.